tests/int/tests/bootstrap.js

```diff
@@ -0,0 +1,40 @@
+const assert = require('chai').assert;
+
+const dbUtils = require('../utils/db'),
+      hortiUtils = require('../utils/horti');
+
+describe('Bootstrapping', () => {
+  before(() =>
+    dbUtils.initBuildsDB()
+      .then(() => dbUtils.uploadBuild(require('./bootstrap/minimal-app-1.0.0.json')))
+      .then(() => dbUtils.uploadBuild(require('./bootstrap/minimal-app-1.1.0.json'))));
+
+  beforeEach(() => dbUtils.initAppsDB());
+
+  it('Bootstraps and installs a minimal application', () => {
+    return Promise.resolve()
+      .then(() => hortiUtils.start([
+        '--install=test:test-app-1:1.0.0',
+        '--no-daemon',
+        '--local'], true, true))
+      .then(() => dbUtils.appDb().get('_design/test-app-1'))
+      .then(ddoc => {
+        assert.equal(ddoc.deploy_info.user, 'horticulturalist cli');
+        assert.equal(ddoc.deploy_info.version, '1.0.0');
+      })
+      .catch(err => assert.fail(err));
```

Comment:
I think if you're returning the promise you don't need this line, but I could very well be wrong...

Reply:
Yep you're absolutely right, I was losing my mind trying to get this test to work out why this test was not doing what I wanted and put that in there to double check something. Will remove.